### PR TITLE
Save and dump last mouse move.

### PR DESCRIPTION
**Pull request by shurymury**
Save and dump last mouse move.
Added Component.hasFocus() to the dump.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Referenced JBS issue must only be used for a single change
- [x] Change must be properly reviewed (no review required)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jemmy-v2 pull/6/head:pull/6` \
`$ git checkout pull/6`

Update a local copy of the PR: \
`$ git checkout pull/6` \
`$ git pull https://git.openjdk.org/jemmy-v2 pull/6/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6`

View PR using the GUI difftool: \
`$ git pr show -t 6`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jemmy-v2/pull/6.diff">https://git.openjdk.org/jemmy-v2/pull/6.diff</a>

</details>

### CODE REVIEW

**Comment by shurymury** [comment]
@lawrence-andrews, can you review?

File: src/org/netbeans/jemmy/drivers/MouseDriver.java [hunk 1]
```diff
@@ -25,13 +25,21 @@
 package org.netbeans.jemmy.drivers;
```
Comment (lawrence-andrew) [inline src/org/netbeans/jemmy/drivers/MouseDriver.java:26]: Copy right 2016 should be modified to 2022

File: test/org/netbeans/jemmy/util/DumpTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @Test
+    public void common() {
+        StringWriter out = new StringWriter();
+        Point point = new Point(1, 2);
+        UIStatus.mouseMoved(fo, point);
+        Dumper.dumpAll(new PrintWriter(out));
+        assertTrue(out.toString()
+                .contains("<property name=\"" + UIStatus.LAST_MOUSE_MOVE_DPROP + "\" value=\""+point+"\"/>"));
+        assertTrue(out.toString()
+                .contains("<property name=\"" + UIStatus.LAST_MOUSE_MOVE_OPERATOR_DPROP + "\" value=\""+fo+"\"/>"));
```
Comment (lawrence-andrew) [inline test/org/netbeans/jemmy/util/DumpTest.java:61]: Can we check whether HAS_FOCUS_DPROP is available in dump?

**Comment by lawrence-andrew** [comment]
Looks good to me.

**Pull request merged**